app-support/access-control.c

```diff
@@ -137,6 +137,7 @@ _ac_timeout()
 int
 _on_ac_notification(const uint8_t* interest, uint32_t interest_size, void* userdata)
 {
+  (void*) userdata;
```

Comment:
I compared this to my NDNph codebase. It seems that you only need `(void)userdata`, not `(void*)userdata`.

Reply:
Changed with the latest commit (f5c8d29)